kms/sshagentkms/sshagentkms_test.go

```diff
@@ -294,7 +294,7 @@ func TestSSHAgentKMS_CreateSigner(t *testing.T) {
 		t.Fatal(err)
 	}
 	block, _ := pem.Decode(b)
-	block.Bytes, err = x509.DecryptPEMBlock(block, []byte("pass")) //nolint
+	block.Bytes, err = x509.DecryptPEMBlock(block, []byte("pass")) // nolint
 	if err != nil {
```

Comment:
Is this `nolint` active at all? `//nolint` is in fact how it's required; otherwise it won't work. But we also usually have to add the specific linter behind it. So I guess this file is currently excluded, likely because it's a test.

Reply:
My IDE must have done this automatically, I didn't even notice. `x509.DecryptPEMBlock` is deprecated as it's insecure, but this is in tests, so 🤷‍♂️

```suggestion
	block.Bytes, err = x509.DecryptPEMBlock(block, []byte("pass"))
	if err != nil {
```

Reply:
Removed it in 6beadb23ed11ededcfe34062db232014921d6fe7.